tests/src/data-generator/integrations.ts

```diff
@@ -34,13 +35,20 @@ const log: any = {
     /*  */
   },
 };
-export const genAppProps = (): RootComponentProps => ({
+export const genAppProps = (): UseRootComponentReturn => ({
```

Comment:
```suggestion
export const genAppProps = (): ReturnType<typeof useRootComponentProps> => ({
```

Reply:
we tried this but it caused some cyclic dependency error, because it would need to have hooks package as dependency, whereas hooks package already imports tests package as deps